tez-api/src/test/java/org/apache/tez/dag/api/TestEntityDescriptor.java

```diff
@@ -69,12 +69,12 @@ public void testSegmentedWrite(InputDescriptor entityDescriptor, InputDescriptor
     entityDescriptor.write(out);
     out.close();
 
-    Mockito.verify(entityDescriptor).writeSegmented(eq(out), any(ByteBuffer.class));
+    verify(entityDescriptor).writeSegmented(eq(out), any());
     deserialized.readFields(new DataInputStream(new ByteArrayInputStream(bos.toByteArray())));
     verifyResults(entityDescriptor, deserialized, payload, confVal);
   }
 
-  @Test (timeout=1000)
+  @Test (timeout=3000)
```

Comment:
is this change needed?

Reply:
I'm not sure. Initially this test timed out with the original timeout value. I double check this one.